src/engine/server/server.h

```diff
@@ -483,4 +483,5 @@ class CServer : public IServer
 #endif
 };
 
+extern CServer *CreateServer();
```

Comment:
```suggestion
CServer *CreateServer();
```

`extern` is superfluous here.

Reply:
I followed literally *all* the others IInterfaces `Create*` signatures:
https://github.com/ddnet/ddnet/blob/1abe73c2ec8b8323102f584229a810d4e8f8c909/src/engine/engine.h#L43-L44
https://github.com/ddnet/ddnet/blob/1abe73c2ec8b8323102f584229a810d4e8f8c909/src/engine/map.h#L42
https://github.com/ddnet/ddnet/blob/1abe73c2ec8b8323102f584229a810d4e8f8c909/src/engine/server.h#L327
https://github.com/ddnet/ddnet/blob/1abe73c2ec8b8323102f584229a810d4e8f8c909/src/engine/console.h#L125
https://github.com/ddnet/ddnet/blob/1abe73c2ec8b8323102f584229a810d4e8f8c909/src/engine/storage.h#L45-L47
https://github.com/ddnet/ddnet/blob/1abe73c2ec8b8323102f584229a810d4e8f8c909/src/engine/config.h#L25
https://github.com/ddnet/ddnet/blob/1abe73c2ec8b8323102f584229a810d4e8f8c909/src/engine/server/antibot.h#L57

IIUC, having those symbols `extern` 'll force compiler to export them when implementation is linked into shared library. Though in that case we'd better explicitly use import/export declarations for compatibility with MSVC.